### DISCUSSION TITLE
PatchTool 6.08 3/25 - fixes for "Official Translation"

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#116] [General]
基本上只修正簡繁轉換，無需求可不更新。

※由於有些解壓縮軟體會略過空資料夾而導致不會建立backup資料夾，故在backup裡面隨便塞了一個檔案應付。

- FFXIVChnTextPatch-0320.rar：基本包。一般使用者下載這個即可。

其他可選：

- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [rawexd-zhcn-0320.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.55/rawexd-zhcn-0320.rar)：簡體中文CSV。解壓後改名為rawexd放到resource資料夾下（取代原資料夾）。
- FFXIV-jar-0211.rar：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0320.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.55/backup-0320.rar)：乾淨的0x0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原，然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.56'>PatchTool 6.08 3/25 - fixes for "Official Translation"</a>.</em>

**Comment by ACZife** [comment]
請問目前用英文介面，但在萬魔殿BOSS技能讀條出現日文是正常的嗎~?

**Reply by GpointChen** [reply]
可能部分技能殘留文字沒刪乾淨，如果有例圖的話可以幫我節省找的時間。

感謝。
